### PR TITLE
Update the content

### PR DESCRIPTION
**Pull request by joinimran**
WHFB has 3 deployment models whereas the document was showing 2. Updated the content.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6168

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-deployment-guide.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Do not begin your deployment until the hosting servers and infrastructure (not r
 
 ## Deployment and trust models
 
-Windows Hello for Business has two deployment models: Hybrid and On-premises. Each deployment model has two trust models: *Key trust* or *certificate trust*.
+Windows Hello for Business has three deployment models: Cloud, Hybrid and, On-premises. Hybrid and On-premises deployment model has two trust models: *Key trust* or *certificate trust*.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-deployment-guide.md]:
```suggestion
Windows Hello for Business has three deployment models: Cloud, Hybrid, and On-premises. Hybrid and On-premises deployment models have two trust models: *Key trust* or *certificate trust*.
```

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**